hazelcast/src/main/java/com/hazelcast/cp/internal/raft/impl/dto/AppendFailureResponse.java

```diff
@@ -76,19 +84,26 @@ public void writeData(ObjectDataOutput out) throws IOException {
         out.writeInt(term);
         out.writeObject(follower);
         out.writeLong(expectedNextIndex);
+        out.writeLong(flowControlSequenceNumber);
     }
 
     @Override
     public void readData(ObjectDataInput in) throws IOException {
         term = in.readInt();
         follower = in.readObject();
         expectedNextIndex = in.readLong();
+        try {
+            flowControlSequenceNumber = in.readLong();
+            // TODO RU_COMPAT_5_3 added for Version 5.3 compatibility. Should be removed at Version 5.5
+        } catch (EOFException e) {
+            flowControlSequenceNumber = 0;
```

Comment:
```suggestion
            flowControlSequenceNumber = -1;
```

Since `flowControlSequenceNumber` starting value is zero, giving it an unexpected value, like -1, here may help to detect unexpected conditions.